src/main/java/org/folio/consortia/controller/FolioTenantController.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.consortia.controller;
+
+import org.folio.consortia.config.FolioExecutionContextHelper;
+import org.folio.spring.controller.TenantController;
+import org.folio.spring.service.TenantService;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.extern.log4j.Log4j2;
+
+@RestController("folioTenantController")
+@RequestMapping
+@Log4j2
+public class FolioTenantController extends TenantController {
```

Comment:
possibly it should be done in FolioTenantService

Reply:
I think so